ostree-simplified-installer.sh

```diff
@@ -336,6 +336,112 @@ greenprint "🧽 Clean up container blueprint and compose"
 sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete container > /dev/null
 
+#################################################################
```

Comment:
Could you please add this kind of test at the end of whole test workflow?
1. Negative test is required, but as what you tested, they are not high priority.
2. Your tests are all about `text matching`, which has more chances to make test flaky.
3. The rest of tests(with high priority) will not run if your test failed.